packages/cliqz-home-react/src/App.js

```diff
@@ -58,6 +66,10 @@ class App extends Component {
   render() {
     return (
       <div id="app">
+        <div id="flexContainer">
+          <MessageCenter position="top" messages={this.state.messages} /> 
```

Comment:
instead of passing position I would rather pass a specific property of messages object. And as probably top and middle are rendered in totally different way, those should be different components (that may inherit from same base class)